### DISCUSSION TITLE
is there any sample code available for web

### DISCUSSION THREAD

**Discussion by vishwastawar** [isar/isar#640] [Help]
I have 12k records and am trying to load that records into the Isar database but it taking too much time (3 seconds) to insert that data. how can I reduce the time for inserting data into it?
thanks.

**Comment by simc** [comment]
Unfortunately IndexedDB performance is lacking sometimes. It's especially bad on Chrome. All other browsers are much faster. That being said, even big apps like WhatsApp use IndexedDB. It just shows a loading bar to the user.
